CHANGELOG-5.X.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+* [RedirectBundle] Allow UTF8 in redirects [#1961](https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/1961)
```

Comment:
Also while you're busy, can you delete this change? This is done automatically when we release a new version.